create_graphics.py

```diff
@@ -230,6 +230,14 @@ def parse_args():
         )
 
     # Short args
+    parser.add_argument(
+        '-r',
+        dest='img_res',
+        default=75,
```

Comment:
Can we keep the default at 72? I'm not sure how sensitive the front-end software is to this number.

Reply:
Oops! I had actually intended to keep the default at 72, but got the wrong number in my head when changing the code.